plans/academic_posse_improvement.markdown

```diff
@@ -0,0 +1,105 @@
+
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need to refine and enhance the posse structure to support mentors in being effective partners with Turing students.
+
+* Why is this a “challenge”?
+
+Posses can "fall apart".
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+I don't think it can be 100 percent solved, the people will always be that "X Factor"
+
+* What happens if we fail to solve it / improve it?
+
+Things will continue as they have, with some people taking advantage of their posse, and some not.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+I didn't feel my posse was that effective, and would have liked it to have been more so.
+
+
+## The Past
+
+* What’s been done so far?
+
+There's a new posse structure, and now we have posse points.
+
+* Was it effective? How do you know?
+
+So far it has been more effective than it has in the past. Posse points is competitive and for some people this brings them together.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+Rachel has been working on this, and continues to do so.
+
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+I've been thinking about the possibility of rotating mentors after each module, to expose more people to the posses.
+
+* What will be the next three things you do?
+
+Possibly schedule assigned intra posse pairing time. Reach out to our mentors. Assign additional mentors to the students who are 
+struggling.
+
+* How will you keep track of progress and results?
+
+We should somehow track how many mentor hours are used each week and possibly how many student pairing hours.
+
+* Are you empowered to take the actions you think necessary?
+
+Like I said in a previous challenge, I don't feel comfortable taking actions solo, but I feel that I could solicit good feedback
+from anyone on the staff.
+
+## The Future
+
+* What would total success look like?
+
+We see people doing things with their posses outside of assigned things.
+
+* What metrics can be used to evaluate results?
+
+An uptick in intra posse pairings.
+
+* What would meaningful progress look like in 3 months? 6 months? 12 months?
+
+I think we would see more posse identity, and personality for better or for worse.
+
+* How feasible are each of those goals?
+
+I tihnk they are rather feasible, since there's great room for improvement.
+
+## The Periphery
+
+* What challenges are you going to stop working on to put time into this?
+
+Tools will probably suffer.
+
+* What research/learning do you need to do to be effective at solving it?
+
+Possibly some way for mentors and students to log who they pair with and for how long.
```

Comment:
I'm not sure the most effective way to handle this. We certainly want it to be as simple as possible for mentors.  Maybe some form they just fill out when they complete a session. We could totally start awarding Mentor Points for filling out the form.